Homework6/Task.py

```diff
@@ -0,0 +1,146 @@
+"""
+Create simulator from real life. This can be
+booking room in hotel, visit to casino, visit to bar.
+Create 3-4 objects, that can describe situation.
+Objects should contain attributes and methods
+to simulate some use cases.
+Completed program should print object states,
+it actions (methods) and objects interaction.
+Every bar can have it's own menu with unique prices
+Each person has his own name, amount of money and
+his own norm in drinking alcohol
+"""
+
+
+class Bar(object):
+    earned = 0
+    name = "Unknown"
+
+    menu_amount = {"beer": 50000,
+                   "whiskey": 500,
+                   "vodka": 1000}
+
+    menu_prices = {"beer": 5,
+                   "whiskey": 10,
+                   "vodka": 3}
+
+    menu_portions = {"beer": 500,
+                     "vodka": 50,
+                     "whiskey": 50}
+
+    def __init__(self, name=name, menu_am=None,
+                 menu_pr=None):
+        self.name = name
+        if menu_am:
+            self.menu_amount = menu_am
+        if menu_pr:
+            self.menu_prices = menu_pr
+
+    def sell(self, drink):
+        self.menu_amount[drink] -= self.menu_portions[drink]
+        self.earned += self.menu_prices[drink]
+
+
+class Person(object):
+    name = "Unknown"
+    cur_bar = None
+    money = 0
+    condition = 0
+    condition_stages = {"almost_ok": 25,
+                        "enough": 50,
+                        "stop": 75,
+                        "overdose": 100}
+    done = False
```

Comment:
as it would be class variable it would cause a lot of troubles when we'll have many persons